app/ide-desktop/lib/dashboard/src/hooks/billing/FeaturesConfiguration.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file
+ *
+ * Paywall configuration for different plans.
+ */
+
+import type * as text from '#/text'
+
+import * as backend from '#/services/Backend'
+
+/**
+ *
+ */
+export const PAYWALL_FEATURES = {
```

Comment:
Why, though? enums have no advantages over regular objects.

Reply:
> similarly: regular objects have no advantages over enums.

It's not true, because objects are:
1. Js-compatible
2. Do not emit extra code in runtime
3. Do not have pitfalls enums have

Reply:
> enums have extra type safety (they are nominally typed)

function some(thing: MyEnum) {}
is not safer than
function some(thing: MyObjType) {}

More details are here: https://www.youtube.com/watch?v=jjMbPt_H3RQ
(I guess it's the best video that describes my experience with enums)